### ISSUE TITLE
Only stands in 1 area

### ISSUE THREAD

**Issue by lolmoney1990** [finndev/PokeBuddy#349]
I have farm Pokestops on, but it only stays in between these same 4 poke stops (causing alot of xp loss/alot of doing nothing). 

Anyway to fix this? Ive been stuck at 300xp farmed for about 6 minutes now (as it doesnt walk around hitting stops)

**Comment by IamDeric** [comment]
Did you check catch pokemon?

**Comment by lolmoney1990** [comment]
Yes I did, would that cause me to not move around far?

**Comment by IamDeric** [comment]
Whats your speed?

**Comment by 404unknown** [comment]
Is there any more pokestop around other than the 4 stops your bot stays at?

**Comment by IamDeric** [comment]
Leave it on. So i will catch pokemon, check your settings also

**Comment by lolmoney1990** [comment]
I use 10km/hr,
It looks like the bot hits 3 pokestops then they all turn blue again (resetting them) and then my guy walks and hit those same ones over again.

**Comment by lolmoney1990** [comment]
I am in an area of about 20+ pokestops lol

**Comment by IamDeric** [comment]
Yes, change your coordinates, like pier 39 ca, bryan park ny, central park, etc

**Comment by IamDeric** [comment]
Try to restart it, or re-download

**Comment by lolmoney1990** [comment]
Done both :(
